src/containers/Application/ApplicationPage.tsx

```diff
@@ -69,16 +71,18 @@ const ApplicationPage: React.FC<ApplicationProps> = ({
   return (
     <Container id="application-form">
       <Grid stackable>
-        <Ref innerRef={contextRef}>
-          <Grid.Column width={4} id="progress-column" className="dev-border">
-            <ProgressArea
-              structure={fullStructure}
-              requestRevalidation={requestRevalidation as MethodRevalidate}
-              strictSectionPage={strictSectionPage as SectionAndPage}
-              context={contextRef}
-            />
-          </Grid.Column>
-        </Ref>
+        {!isMobile && (
```

Comment:
This was already being hidden using the `hide-on-mobile` class, but I figured it's probably better to prevent rendering of the entire component on mobile, as it's quite a complex one.

(Am I correct in thinking that if it's hidden with a class then React will still render it?)

Reply:
Yes, it still gets rendered but with display=none.  So that is better really.